src/Workspaces/CSharp/Portable/CodeStyle/CSharpCodeStyleOptions.cs

```diff
@@ -1,61 +1,87 @@
 // Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
-using System.Collections;
 using System.Collections.Generic;
 using Microsoft.CodeAnalysis.CodeStyle;
 using Microsoft.CodeAnalysis.Options;
+using static Microsoft.CodeAnalysis.CodeStyle.CodeStyleHelpers;
 
 namespace Microsoft.CodeAnalysis.CSharp.CodeStyle
 {
     internal static class CSharpCodeStyleOptions
     {
         // TODO: get sign off on public api changes.
         public static readonly Option<bool> UseVarWhenDeclaringLocals = new Option<bool>(nameof(CodeStyleOptions), nameof(UseVarWhenDeclaringLocals), defaultValue: true,
-            storageLocations: new RoamingProfileStorageLocation("TextEditor.CSharp.Specific.UseVarWhenDeclaringLocals"));
+            storageLocations: new OptionStorageLocation[]{
```

Comment:
params array doesn't work with named arguments.

It can either look like this:
```C#
        public static readonly Option<bool> UseVarWhenDeclaringLocals = new Option<bool>(nameof(CodeStyleOptions), nameof(UseVarWhenDeclaringLocals), defaultValue: true,
            storageLocations: new OptionStorageLocation[]{
                new EditorConfigStorageLocation("csharp_style_var_for_locals", ParseEditorConfigCodeStyleOption),
                new RoamingProfileStorageLocation("TextEditor.CSharp.Specific.UseVarWhenDeclaringLocals")});
```

Or this: 
```C#
        public static readonly Option<bool> UseVarWhenDeclaringLocals = new Option<bool>(nameof(CodeStyleOptions), nameof(UseVarWhenDeclaringLocals), true,
                new EditorConfigStorageLocation("csharp_style_var_for_locals", ParseEditorConfigCodeStyleOption),
                new RoamingProfileStorageLocation("TextEditor.CSharp.Specific.UseVarWhenDeclaringLocals"));
```